src/css/sections/lessons.css

```diff
@@ -0,0 +1,74 @@
+.lessons-list-item {
+  border: 1px solid var(--green-color);
+  border-radius: 16px;
+  padding: 40px 20px;
+  min-height: 400px;
+  max-width: 335px;
+}
+@media only screen and (min-width: 768px) {
+  .lessons-list-item {
+    border-radius: 30px;
+    padding: 32px 16px;
+    min-height: 404px;
+    max-width: 336px;
+  }
+}
+@media only screen and (min-width: 1152px) {
+  .lessons-list-item {
+    padding: 32px;
+    max-width: 320px;
+  }
+}
+.lessons-list-item.accent {
+  color: var(--white-color);
+  background-color: var(--green-color);
+}
+.lessons-list-item.accent .secondary-text {
+  color: rgba(var(--white-color-rgb), 0.7);
+}
+.lessons-list-item .secondary-text {
+  flex-grow: 1;
+  margin-bottom: unset;
+}
+@media only screen and (min-width: 768px) {
+  .lessons-list-item .secondary-text {
+    font-size: 16px;
+    line-height: 1.5;
+  }
+}
+.lessons-list-item .btn {
+  max-width: 295px;
```

Comment:
I think button should just be on the full width of container. Maybe display: block is enough here

Reply:
updated it to 
.lessons-list-item .btn {
  width: 100%;
}
now buttons size is as expected